sdk/scheduler/src/test/java/com/mesosphere/sdk/http/queries/PodQueriesTest.java

```diff
@@ -69,30 +69,30 @@
                 .setType("test")
                 .setIndex(0)
                 .toProto());
-        POD_0_TASK_A = infoBuilder.setName("test-0-a").setTaskId(CommonIdUtils.toTaskId("a")).build();
+        POD_0_TASK_A = infoBuilder.setName("test-0-a").setTaskId(CommonIdUtils.toTaskId(TestConstants.SERVICE_NAME, "a")).build();
```

Comment:
Do we have a `TestIdUtils` class where we could have a `getTaskId(String taskName)` method? Not critical though.